src/components/code-block.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import '../styles/components/code-block.scss';
+
+const CodeBlock = ({ lightMode, children, className, ...props }) => (
+  <pre
+    className={`codeblock${lightMode ? ' codeblock-light' : ''}${
+      className ? ` ${className}` : ''
+    }`}
+    {...props}
+  >
+    <code>{children}</code>
+  </pre>
+);
+
+CodeBlock.propTypes = {
+  lightMode: PropTypes.bool,
```

Comment:
Adding /** comments */ to proptypes is then picked up by storybook to display the doc. See other components.

Reply:
Ah, I hadn't realise it was used. I added comments now then.